src/components/chip/examples/chip-link.tsx

```diff
@@ -0,0 +1,42 @@
+import { Component, h } from '@stencil/core';
+
+/**
+ * Chip as hyperlink
+ * A chip can be used to navigate to a different page, pane of the application
+ * or to an external site. To get the chip to act as a hyperlink, you need to
+ * provide a `Link`.
+ *
+ * This way, the component will automatically become a native HTML link.
+ * This gives the users the possibility of interacting with links
+ * in a natural way, for instance they can open any of the previous
+ * pages in a new browser tab. This can also have other accessibility benefits.
+ *
+ * :::note
+ * Clicking a link will open it in current window by default,
+ * and this reloads the entire webpage.
+ * To avoid reloading the whole application (in the context of a single-page apps),
+ * you might want to handle the navigation with your application's router,
+ * :::
```

Comment:
I don't know if we need this note here. I don't think it's going to help anyone. For anyone building stuff inside Lime CRM, the navigation interceptor should take care of this for them, and for anyone building their own app, if they don't know they need to consider this, talking about using the application's router probably isn't going to help them…

Reply:
Ok. This was also a copy/paste form other components too (which I think you and I did together actually 😅 ). But I think saying it is not bad, to remind the consumer what to think about, when designing interactions.

Reply:
I removed it